ajax/libs/concrete/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "concrete",
+  "version": "0.1.0",
+  "filename": "concrete.min.js",
+  "description": "Concrete is a lightweight Html5 Canvas framework that enables hit detection, layer support, pixel ratio management, exports, and downloads.",
+  "keywords": [
+    "concrete",
+    "html5",
+    "canvas",
+    "2d",
+    "graphics",
+    "javascript"
+  ],
+  "homepage": "http://www.concretejs.com",
+  "author": {
+    "name": "Eric Rowell",
+    "url": "https://twitter.com/ericdrowell"
+  },
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ericdrowell/concrete"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/ericdrowell/concrete.git",
+    "basePath": "build/",
+    "files": [
+      "concrete.js",
+      "concrete.min.js"
```

Comment:
If `build/` folder is for distribution, would you please change the files to only single rule `"**/*"` so that if you added more files in it, we don't need to update this config? Thanks.